x/observer/types/errors.go

```diff
@@ -37,4 +34,5 @@ var (
 	ErrInboundDisabled      = errorsmod.Register(ModuleName, 1132, "inbound tx processing is disabled")
 	ErrInvalidZetaCoinTypes = errorsmod.Register(ModuleName, 1133, "invalid zeta coin types")
 	ErrNotObserver          = errorsmod.Register(ModuleName, 1134, "sender is not an observer")
+	ErrInvalidBlockHeader   = errorsmod.Register(ModuleName, 1135, "invalid block header")
```

Comment:
imo , Errors related to a module should be defined in that module itself, even if other modules use them. 
Right now, only the observer module is using this, but in the future, we might use this in other places 

 we should be doing this for other modules as well , might be a good idea to create a task to trackit

Reply:
5c5bc0d5